java/src/test/android/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
do we need to check in gradlew and gradlew.bat?
I noticed in the top level .gitignore file they are excluded from the java/ directory.
if we don't need them, perhaps we can also update that .gitignore file.

Reply:
No, we don't need gradlew and gradlew.bat. will update.